utils/test_http_server/http_server.py

```diff
@@ -0,0 +1,36 @@
+import ipaddress
+import http_server_test_funcs as test
+import sys
+
+#   IF YOU WANT TO ADD A TEST
+#       add test data in server_tests/tests.json with JSON syntax
+#       check if current test implementation will test it properly:
+#           http_server_test_funcs.py > send_request()  - if JSON request, dont add content-type, it is added automatically
+#                                       test_response() 
+
+def main():
+    if test_argv() is 0:
```

Comment:
what does this line do?

Reply:
test if you gave the sctipt a proper ip address arg

Reply:
you have to use argparse